### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by minho42**

### CODE REVIEW

**Comment by bertjwregeer** [comment]
Wow, this is amazing! Thanks!

I'll let @stevepiercy take it from here!

File: BFG_HISTORY.rst [hunk 1]
```diff
@@ -1670,7 +1670,7 @@ Internal
 - Remove the ``repoze.bfg.testing.registerTraverser`` function.  This
   function was never an API.
 
-Documenation
+Documentation
 ------------
```
Comment (stevepiercy) [inline BFG_HISTORY.rst]: Underline length must match length of text.

File: BFG_HISTORY.rst [hunk 2]
```diff
@@ -2131,7 +2131,7 @@ Internals
   current ``repoze.bfg`` registry.  They fall back to obtaining the
   registry from the threadlocal API.
 
-Backwards Incompatibilites
+Backwards Incompatibilities
 --------------------------
```
Comment (stevepiercy) [inline BFG_HISTORY.rst]: Underline length must match length of text.

**Comment by minho42** [comment]
@stevepiercy
Done. 
Thanks.

**Comment by stevepiercy** [comment]
And we now have a successful build! Thank you for your corrections.

**Pull request merged**